projects/pgai/db/sql/incremental/030-semantic-catalog.sql

```diff
@@ -0,0 +1,13 @@
+
+create table ai.semantic_catalog
```

Comment:
can we record the names of the created tables into this row? if we ever change the naming convention this will be useful.